File: program/program/find-the-largest-three-elements-in-an-array/find_the_largest_three_elements_in_an_array.php
<?php
$input = array(10, 4, 3, 50, 23, 90);

if(count($input) < 3 ) {
    echo "Array should have a minimum of 3 elements";
} else {
rsort($input);
$arrlength = count($input); 
echo $input[0].",".$input[1].",".$input[2];
}
?>

File: program/program/find-the-largest-three-elements-in-an-array/FindTheLargestThreeElementsInAnArray.rb
input = [12, 13, 1, 10, 34, 1];
if input.length() <= 2
  print "Array should contain atleast 3 elements"
else
  sort = input.sort.reverse
  result = "#{sort[0].to_i},#{sort[1].to_i},#{sort[2].to_i}"
  print result
end